index.js

```diff
@@ -91,6 +91,7 @@ function toggleSound(){
 
     // unset image src when either card flip or show all cards
     const unsetImgSource = (card) =>{
+        flip(card)
         card.children[0].src="#"; // Removing image src so that it isn't visible through HTML
         card.children[0].alt="card front face"; // Removing image alt so that it isn't visible through HTML
         card.children[1].style.display="block";

```

Comment:
hiding/unhiding the front pattern was temporary till this issue. Now after adding the rotation, there should be no need to hide/unhide it, as it should go backside.

so remove that